docs/source/getting-started/quickstart.rst

```diff
@@ -97,9 +97,19 @@ Let's apply Laplace noise to a value.
             :end-before: /demo
 
 This is obviously not the easiest way to add noise to a number,
-but it demonstrates a number of OpenDP patterns:
+but it demonstrates a number of low-level OpenDP patterns:
 
-* Defining your metric space with ``space_of`` in Python's Context API, or a (domain, distance) tuple in any language..
-* Chaining operators together with ``>>`` in Python and Rust, or ``|>`` in R.
-* Constructing a ``Measurement`` function on your metric space with ``then_laplace``.
-* Invoking that measurement on a value to get a DP release.
+* First, define your metric space.
```

Comment:
Your prose is better, as it talks about the concepts and not the syntax.

Maybe another tweak though:
```suggestion
* First, define your metric space (your data domain and definition of neighboring)
```

Reply:
"definition of neighboring" is kind of clunk... Let me hold off on this and re-read the whole thing with updates.

Reply:
Ended up with 
> First, define your “metric space”: a data domain and a definition of distance